### PR TITLE
QPPA-516 Use qualityId as measureId

### PR DESCRIPTION
**Pull request by sungchoi**
### Description:
We are updating quality measurement measureIds to be equal to their qualityIds.

From JIRA Ticket:
The QPP API for quality data created a measureId that is a concatenation of qualityId and other ID fields. This field is NOT useful so we'd like to, for quality measures, remove the qualityId field and replace the existing measureId with what is now the qualityId.

This includes making follow up changes in benchmarks.

Update submissions API since historical benchmarks are keyed by measureId.

### Automated Test Plan

- [x] `npm test` should be green
- [x] `cat measures/measures-data.json | node scripts/validate-data.js measures ` should return `Valid!`
- [x] `cat benchmarks/2017.json | node scripts/validate-data.js benchmarks` should return `Valid!`

### Manual Test Plan

- [x] Code Review
- [x] `measures/measures-data.json` should be updated
- [x] `benchmarks/2017.json` should be updated

### CODE REVIEW

File: scripts/convert-qpp-to-measures.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ process.stdin.on('end', () => {
  *    emsr_id               | eMeasureId
  *    nqf_emsr_num          | nqfEMeasureId
  *    nqf_num               | nqfId
- *    qlty_id               | qualityId
+ *    qlty_id               |
```
Comment (abarciauskas-bgse) [inline scripts/convert-qpp-to-measures.js]: Hmm maybe this should be removed altogether - are there other fields we don't use?

Comment (marimiyachi) [inline scripts/convert-qpp-to-measures.js]: the measure_id row should be deleted an this should map qlty_id to measureId

File: scripts/convert-qpp-to-measures.js [hunk 2]
```diff
@@ -142,69 +142,72 @@ function parseQpp(json) {
...
+      } else if (key === 'measure_type') {
+        obj.measureType = formatString(value);
+      } else if (key === 'emsr_id') {
+        obj.eMeasureId = parseId(value);
+      } else if (key === 'nqf_emsr_num') {
+        obj.nqfEMeasureId = parseId(value);
+      } else if (key === 'nqf_num') {
+        obj.nqfId = parseId(value);
+      } else if (key === 'qlty_id') {
+        obj.qualityId = parseId(value);
```
Comment (marimiyachi) [inline scripts/convert-qpp-to-measures.js]: This should be removed.

File: benchmarks/benchmarks-schema.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ definitions:
     type: object
     properties:
       measureId:
-        description: measureId corresponds to a measure object's measureId
+        description: measureId corresponds to a measure object's measureId. For quality measures, measureId is the same as qualityId.
```
Comment (marimiyachi) [inline benchmarks/benchmarks-schema.yaml]: I think this line is not necessary - all references to qualityId should be removed.

File: scripts/convert-qpp-to-measures.js (hunk 1 above)
Comment (sungchoi) [inline scripts/convert-qpp-to-measures.js]: Aha! That makes sense.

File: benchmarks/benchmarks-schema.yaml (hunk 3 above)
Comment (sungchoi) [inline benchmarks/benchmarks-schema.yaml]: Got it.

File: scripts/convert-qpp-to-measures.js (hunk 2 above)
Comment (sungchoi) [inline scripts/convert-qpp-to-measures.js]: Good catch.

**Pull request closed without merging**